TotalCrossVM/src/nm/ui/GraphicsPrimitives.h

```diff
@@ -20,10 +20,8 @@
  #include "darwin/gfx_ex.h"
 #elif defined(ANDROID)
  #include "android/gfx_ex.h"
-#elif defined(linux)
- #ifdef TOTALCROSS
-  #include "linux/gfx_ex.h"
- #endif
+#elif defined(linux) || defined HEADLESS
```

Comment:
defined(linux) || defined HEADLESS && !darwin

unfortunately iOS build has the flag linux